opsdroid/connector/matrix/connector.py

```diff
@@ -177,6 +201,19 @@ async def connect(self):
 
         self.connection.sync_token = response.next_batch
 
+        if self._allow_encryption:
```

Comment:
Does this break if encryption is disabled? If not can we just continue calling it?

Reply:
i dont think it breaks anything

Reply:
We might as well save the if then?